code/ai/aicode.cpp

```diff
@@ -14430,6 +14438,8 @@ void init_aip_from_class_and_profile(ai_info *aip, ai_class *aicp, ai_profile_t
 		profile->chance_to_use_missiles_on_plr[Game_skill_level] : aicp->ai_chance_to_use_missiles_on_plr[Game_skill_level];
 	aip->ai_max_aim_update_delay = (aicp->ai_max_aim_update_delay[Game_skill_level] == FLT_MIN) ? 
 		profile->max_aim_update_delay[Game_skill_level] : aicp->ai_max_aim_update_delay[Game_skill_level];
+	aip->ai_smart_secondary_evade_chance = (aicp->ai_smart_secondary_evade_chance[Game_skill_level] == FLT_MIN) ? 
+		profile->max_aim_update_delay[Game_skill_level] : aicp->ai_smart_secondary_evade_chance[Game_skill_level];
```

Comment:
Looks like a copy+paste typo here.  Shouldn't the field after the question mark be a secondary_evade_chance?